flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/KubernetesOperatorConfigOptions.java

```diff
@@ -203,4 +203,11 @@ public class KubernetesOperatorConfigOptions {
                     .defaultValue(false)
                     .withDescription(
                             "Enables dynamic change of watched/monitored namespaces. Defaults to false");
+
+    public static final ConfigOption<Boolean> OPERATOR_JOSDK_METRICS_ENABLED =
+            ConfigOptions.key("kubernetes.operator.josdk.metrics.enabled")
+                    .booleanType()
+                    .defaultValue(true)
+                    .withDescription(
+                            "Enables that Metrics of Java Operator SDK forwards metrics to the Flink metric registries . Defaults to true");
```

Comment:
Please do not include the default value in the description. It will be part of the HTML generated

Reply:
@gyfora , config options like `kubernetes.operator.watched.namespaces`, `kubernetes.operator.dynamic.namespaces.enabled` include the default value in the description. Do I need to update these description of these options?

Reply:
Don't worry about the others for now, Matyas will fix some of those